lib/popup.ts

```diff
@@ -180,6 +180,9 @@ export class PopupControl<T extends IPopupOptions = IPopupOptions> extends Dispo
             dom.onElem(triggerElem, 'mouseenter', () => this.open());
             dom.onElem(triggerElem, 'mouseleave', () => this.close());
             break;
+          case 'keydown':
+            dom.onElem(triggerElem, 'keydown', () => this.open());
```

Comment:
What do you think of `(ev) => ev.key === 'Escape' ? this.close() : this.open()` ? Is that a general enough desired behavior for triggers that react to keydown? I am slightly doubtful that we want for `open` to happen on every keydown actually. Maybe that's better to leave to higher-level code than a generic `popup`? I think for an INPUT, you want to react to ENTER/Escape/Up/Down, and anything that actually changes the input (i.e. 'input' event), but not on SHIFT for example, or COPY.

Reply:
I wrote the UserManager input button code in parallel to these changes, and I do have the menu only appear when the email is valid (while "opening" on keydown). Maybe we want something even more targeted but none of the existing triggers fit the use case. At one point I attempted to create a way to open the menu without a trigger by exposing an open function, I abandoned it because it was becoming a bigger change but I could try that again.